velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -417,18 +429,39 @@ class NonNumericMinMaxAggregateBase : public exec::Aggregate {
       if (decoded.isNullAt(i)) {
         return;
       }
+
+      if (shouldCheckNestedNull_) {
```

Comment:
It looks like checkNestedNull and decoded.isNullAt(i) checks always happen together. Consider introducing a helper function: bool checkNulls(decoded, i) that combines these 2 pieces of logic:

```
bool checkNulls(decoded, i) {
   if (decoded.isNullAt(i)) {
        return;
      }

 if (throwOnNestedNulls) {
     VELOX_USER_CHECK...
}
}
```